cloudflare/resource_cloudflare_zone_dnssec.go

```diff
@@ -0,0 +1,156 @@
+package cloudflare
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+// The supported status for a Zone DNSSEC setting
+const (
+	DNSSECStatusActive   = "active"
+	DNSSECStatusDisabled = "disabled"
+)
+
+func resourceCloudflareZoneDNSSEC() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceCloudflareZoneDNSSECCreate,
+		Read:   resourceCloudflareZoneDNSSECRead,
+		Update: resourceCloudflareZoneDNSSECUpdate,
+		Delete: resourceCloudflareZoneDNSSECDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"zoneid": {
```

Comment:
Similar to https://github.com/cloudflare/terraform-provider-cloudflare/pull/852/files#r517643694, we should keep this consistent.

Reply:
fixed